### PR TITLE
Widget now looks like figma design

### PR DESCRIPTION
**Pull request by adriawh**
Added: 
- Looks like the Figma design for light and dark mode
- Adjusts to different system font sizes
- Added the two supported families(widget sizes) medium and small
- Shows the correct icon for transport mode
- Shows day of departure if the departure is in a future day

### CODE REVIEW

**Comment by adriawh** [comment]
Closes https://github.com/AtB-AS/kundevendt/issues/2589

File: ios/departureWidget/Views/SmallView.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(Color("LineInformationTextColor"))
+            }
+            Spacer()
+
+            HStack {
+                ForEach(viewModel.departures(numberOfdepartures: 2), id: \.self) { time in
+                    DepartureTileView(date: time)
+                }
+            }.scaledToFit().minimumScaleFactor(0.1)
```
Comment (jorelosorio) [inline ios/departureWidget/Views/SmallView.swift:28]: I think we could reuse the content inside the body in another view that could be reused in SmallView.swift and MediumView.swift seems pretty similar.

**Review by jorelosorio — CHANGES_REQUESTED** [review]

File: ios/departureWidget/Views/SmallView.swift (hunk 1 above)
Comment (adriawh) [reply ios/departureWidget/Views/SmallView.swift:28]: yes, agreed. Will fix

**Review by jorelosorio — APPROVED** [review]

**Comment by jorelosorio** [comment]
Good work @adriawh the widget looks nice :)

**Pull request merged**